### PR TITLE
feat: equipment schema added, division schema extended

### PR DESCRIPTION
**Pull request by SergeyTatarenko**

### CODE REVIEW

**Comment by SergeyTatarenko** [comment]
Modification required for the new Division page in UI. Feature described in https://zube.io/edenlabllc/e-health/c/7911

File: __schema__/equipment.graphql [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+"""
+input EquipmentFilter {
+  "Equipment type."
+  type: String
+}
+
+"""
+Methods to use when ordering `Equipment`.
+"""
+enum EquipmentOrderBy {
```
Comment (ephor) [inline __schema__/equipment.graphql]: Let's remove this sorting

Comment (kiurchv) [inline __schema__/equipment.graphql]: There are missing default sorting values: `INSERTED_AT_ASC` and `INSERTED_AT_DESC`

Comment (SergeyTatarenko) [reply __schema__/equipment.graphql]: Sorting by external_id replaced with default sorting values: INSERTED_AT_ASC and INSERTED_AT_DESC

Comment (SergeyTatarenko) [reply __schema__/equipment.graphql]: Done

File: __schema__/equipment.graphql [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  "Equipment serial number"
+  serialNumber: String
+  "Note for the equipment"
+  note: String
+  "Flag whether equipment is actual or not"
+  isActive: Boolean
+  "Date and time when record was inserted"
+  insertedAt: DateTime
+  "Date and time when record was updated"
+  updatedAt: DateTime
```
Comment (andriy-ilin) [inline __schema__/equipment.graphql]: is it required fields or not?

Comment (SergeyTatarenko) [reply __schema__/equipment.graphql]: InsertedAt and updatedAt are not required for the query

Comment (kiurchv) [reply __schema__/equipment.graphql]: Yes, since these fields always present they should be required

Comment (SergeyTatarenko) [reply __schema__/equipment.graphql]: Done

**Pull request merged**